### PR TITLE
SPCTR-3180: Fix: CTA - Resolved alignment issue for buttons in call to action.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Additional button and Button is not center aligned in the mobile view buttons goes to right.

### Screenshots

https://d.pr/v/Swny4J

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: includes/blocks/call-to-action/frontend.css.php [hunk 1]
```diff
@@ -266,7 +266,8 @@
 		'column-gap' => UAGB_Helper::get_css_value( $attr['gapBtnTablet'], 'px' ),
 	);
 	$m_selectors[' .uagb-cta__buttons'] = array(
-		'column-gap' => UAGB_Helper::get_css_value( $attr['gapBtnMobile'], 'px' ),
+		'column-gap'   => UAGB_Helper::get_css_value( $attr['gapBtnMobile'], 'px' ),
+		'margin-right' => '40%',
```
Comment (AbhijitNage123) [inline includes/blocks/call-to-action/frontend.css.php:270]: Instead of margin can we try adding `justify-content: center` to fix this issue

Comment (rajmohanreddy774) [reply includes/blocks/call-to-action/frontend.css.php]: Yes, it is working with justify-content : center tested.

**Comment by AbhijitNage123** [comment]
@rajmohanreddy774 is it possible to add `justify-content: center` from Style.scss instead of adding it from dynamic style. Make sure to test everything

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -216,6 +216,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ## Changelog ##
 
 ### x.x.x ###
+* Fix: Call To Action - Resolved an issue wherein the alignment of buttons is not working as intended.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Fix: Call To Action - Resolved an issue wherein the buttons would not be aligned as intended.
```

File: includes/blocks/call-to-action/frontend.css.php [hunk 3]
```diff
@@ -266,7 +266,8 @@
 		'column-gap' => UAGB_Helper::get_css_value( $attr['gapBtnTablet'], 'px' ),
 	);
 	$m_selectors[' .uagb-cta__buttons'] = array(
-		'column-gap' => UAGB_Helper::get_css_value( $attr['gapBtnMobile'], 'px' ),
+		'column-gap'      => UAGB_Helper::get_css_value( $attr['gapBtnMobile'], 'px' ),
+		'justify-content' => 'center',
```
Comment (bsf-zanev) [inline includes/blocks/call-to-action/frontend.css.php:270]: Do we need to add Justify Content - Cente only for the mobile view?

Comment (rajmohanreddy774) [reply includes/blocks/call-to-action/frontend.css.php:270]: Yes in Desktop view if I use Justify-Content and used overall alignment left buttons going out of the wrapper so I have used only with mobile were the issue present.

File: README.md (hunk 2 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

**Review by bsf-zanev — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**